spec/unit/coinbase/transfer_spec.rb

```diff
@@ -299,25 +297,23 @@
 
       before do
         transfer.transaction.sign(from_key)
-        allow(client)
-          .to receive(:eth_getTransactionByHash)
-          .with(transfer.transaction_hash)
-          .and_return(onchain_transaction)
-        allow(client)
-          .to receive(:eth_getTransactionReceipt)
-          .with(transfer.transaction_hash)
-          .and_return(transaction_receipt)
+        allow(transfers_api)
+          .to receive(:get_transfer)
+          .with(transfer.wallet_id, transfer.from_address_id, transfer.id)
+          .and_return(failed_broadcast_model)
       end
 
       it 'returns FAILED' do
-        expect(transfer.status).to eq(Coinbase::Transfer::Status::FAILED)
+        status = transfer.reload.status
+        expect(status).to eq(Coinbase::Transfer::Status::FAILED.to_s)
+        expect(transfer.status).to eq(Coinbase::Transfer::Status::FAILED.to_s)
```

Comment:
This seems superfluous.

We can just test the following:
```
        expect(transfer.reload.status).to eq(Coinbase::Transfer::Status::FAILED.to_s)
```

It could be worth it to test that it reloads other properties as well, but these can just be one-liners